mitmproxy/tools/console/common.py

```diff
@@ -415,14 +501,15 @@ def raw_format_flow(f):
     return urwid.Pile(pile)
 
 
-def format_flow(f, focus, extended=False, hostheader=False, cols=False):
+def format_flow(f, focus, extended=False, hostheader=False, cols=False, layout='default'):
     acked = False
     if f.reply and f.reply.state == "committed":
         acked = True
     d = dict(
         focus=focus,
         extended=extended,
-        two_line=extended or cols < 100,
+        two_line=extended or cols < 80,
```

Comment:
I'm not sure 80 characters is enough to fit the content in the tabular view nicely.

![image](https://user-images.githubusercontent.com/9346237/64911803-fec32e00-d715-11e9-94ea-f876db36ea5a.png)

Reply:
I agree 80 is too low upped the value back to 100.